include/ccmath/internal/support/directional_round.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "ccmath/math/nearest/ceil.hpp"
+#include "ccmath/math/nearest/floor.hpp"
+
+namespace ccm::support {
+	template <typename T, std::enable_if_t<std::is_floating_point_v<T>, bool> = true>
+	constexpr toward_zero(T x) noexcept
```

Comment:
A short doxygen description of what these functions are doing would be nice? Though if you don't feel like it then its fine.